hack/setup_prow_env.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# This file should be sourced in a wrapper script (i.e apply_folder.sh)
+
+echo
+echo "[INFO] Running in OpenShift CI (Prow)."
+echo "[INFO] Setting up for PROW"
+echo
+
+# Validate some mandatory variables
+if [ -z "${ART_NAME:-}" ]; then
+    error_exit "missing mandatory variable \$ART_NAME"
+fi
+
+if [ -z "${TF_FOLDER:-}" ]; then
+    error_exit "missing mandatory variable \$TF_FOLDER"
+fi
+
+if [ -z "${TF_VARS:-}" ]; then
+    error_exit "missing mandatory variable \$TF_VARS"
+fi
+
+OCM_TOKEN=$(cat "${CLUSTER_PROFILE_DIR}/ocm-token")
+if [ -z "${OCM_TOKEN:-}" ]; then
+    error_exit "missing mandatory variable \$OCM_TOKEN"
+fi
+
+# ACCOUNT_ROLE_PREFIX=${ACCOUNT_ROLE_PREFIX:-$NAMESPACE}
+    CLOUD_PROVIDER_REGION=${LEASED_RESOURCE}
+
+# Configure aws
+AWSCRED="${CLUSTER_PROFILE_DIR}/.awscred"
+if [[ -f "${AWSCRED}" ]]; then
+    export AWS_SHARED_CREDENTIALS_FILE="${AWSCRED}"
+    export AWS_DEFAULT_REGION="${CLOUD_PROVIDER_REGION}"
+else
+    error_exit "Did not find compatible cloud provider cluster_profile"
+fi
+
+export TF_VAR_token=${OCM_TOKEN}
+
+WORK_DIR=${SHARED_DIR}/work
+STATE_ARCHIVE="${SHARED_DIR}/${ART_NAME}.tar.gz" 
+
+rm -rf "${WORK_DIR}"
+mkdir "${WORK_DIR}"
+
+if [[ -f "${STATE_ARCHIVE}" ]]; then
+    echo "[INFO] Found TF state archive for '${ART_NAME}' @ ${STATE_ARCHIVE}"
+    tar xvfz "${SHARED_DIR}/${ART_NAME}.tar.gz" -C "${WORK_DIR}"
+fi
+
+export TERRAFORM_D_DIR=/root # location of .terraform.d folder
```

Comment:
how this envar comes to play in terraform init/apply.. you need to change the $HOME in order to take the .terraform.d contents (created by docker build install)

Reply:
Look at the line that executes terraform. It is passed to it.